_data/getting_started.yml

```diff
@@ -7,11 +9,17 @@
       title: 'npm'
       code: 'npm install basement'
 - title: 'Requirements'
+  sub_nav_link: true
+  sub_nav_link: ''
```

Comment:
What's this for?

Reply:
@conordavidson I have to remove `line 13`, but `sub_nav_link` decides whether this section should have a Sub Nav Item on the nav bar. For example, the section on the `Settings` page doesn't appear on the Sub Nav of `Settings`